src/main/java/frc/robot/subsystems/climberSubsystem/ClimberCommandFactory.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.subsystems.climberSubsystem;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.Constants.ClimberConstants;
+import java.util.function.BooleanSupplier;
+import java.util.function.Consumer;
+
+public class ClimberCommandFactory {
+  private final ClimberSubsystem subsystem;
+  private final Runnable noop = () -> {};
+
+  public ClimberCommandFactory(ClimberSubsystem subsystem) {
+    this.subsystem = subsystem;
+  }
+
+  public Command raise() {
+    if (subsystem == null) return new InstantCommand();
+    Timer minTime = new Timer();
```

Comment:
Why are we not using waitSeconds?

https://github.com/frc6377/crescendo_2024/pull/171/files/80a13570e47583dc8d8d327148c797a9186cb59a#r1501015849
https://docs.wpilib.org/en/stable/docs/software/commandbased/commands.html#waiting

Reply:
Different appoarch to same problem